app/models/restaurant.rb

```diff
@@ -13,7 +13,10 @@ def self.for_markers
     all.to_a.map(&:serializable_hash).map do |restaurant|
       {name: restaurant['name'],
        latitude: restaurant['latitude'],
-       longitude: restaurant['longitude']}
+       longitude: restaurant['longitude'],
+       description: restaurant['description'],
+       restaurant_category: restaurant['restaurant_category'],
+       id: restaurant['id']}
```

Comment:
Why do you want to expose the id number?

Reply:
It's so we can go to that route trough the infowindow on the markers. We tried to add a path for it but could not get it to work any other way.